test/micro/org/openjdk/bench/java/io/FileOpen.java

```diff
@@ -85,4 +91,48 @@ public boolean booleanAttributes() {
                 && tmp.isDirectory()
                 && tmp.isFile();
     }
+
+    /**
+     * Examine overheads of converting Files to Paths
+     */
+    @BenchmarkMode(Mode.AverageTime)
+    @OutputTimeUnit(TimeUnit.NANOSECONDS)
+    @State(Scope.Thread)
+    @Warmup(time=2, iterations=5)
+    @Measurement(time=3, iterations=5)
+    @Fork(value=2, jvmArgs="-Xmx1g")
+    public static class ToPath {
```

Comment:
Do we really have to nest this test? It seems to gain nothing per se: the annotations and fields are still pretty much duplicated.

Reply:
Yeah, no. Nesting was to pre-emptively decouple the micros, but it makes sense to squash them into one for now.